src/Packages/Passport/Runtime/ThirdParty/Gree/Assets/Plugins/GreeBrowserClient.cs

```diff
@@ -20,7 +21,12 @@ public GreeBrowserClient()
             PassportLogger.Warn("Native Android and iOS WebViews cannot run in the Editor, so the macOS WebView is currently used to save your development time." +
                 " Testing your game on an actual device or emulator is recommended to ensure proper functionality.");
 #endif
+#if UNITY_WEBGL
+            GameObject webViewGameObject = new GameObject("WebViewObject");
```

Comment:
I tried, but webgl needs it to be a GameObject so that it can communicate with `unityInstance.SendMessage(GameObjectName, "CallFromJS", message);` unityInstance.SendMessage will look for GameObjectName to send the message.